### DISCUSSION TITLE
Let's give stars each other.

### DISCUSSION THREAD

**Discussion by basitali717** [Finalgoal231/finalGoal#98] [About github] [answered]
https://github.com/shinevue/finalGoal
https://github.com/popstar7/finalGoal
https://github.com/techietrend/finalGoal
https://github.com/shiny7star/finalGoal
https://github.com/goldsunshines/finalGoal
https://github.com/silvershiny/finalGoal
https://github.com/EmersonCosta0915/finalGoal
https://github.com/shinyangels/finalgoal
https://github.com/Passionatedev724/finalgoal
https://github.com/Richdev0724/finalgoal
https://github.com/Hotshinydev/finalgoal
https://github.com/Guilmola724/finalgoal
https://github.com/amil3955/finalgoal
https://github.com/robintech514/finalgoal
https://github.com/danilodavid521/finalgoal
https://github.com/vit718/finalgoal
https://github.com/lucatosc/finalgoal

**Comment by noahstoner** [comment, marked as answer]
I have given all stars

**Reply by creative2113** [reply]
I have given all stars too.
